packages/styles/src/link/index.css

```diff
@@ -0,0 +1,87 @@
+/**
+ * Link styles
+ * @variant {small | medium | large} syn-link The size of the link
+ * @boolean { false } syn-link--quiet Applies the quiet link styling
+ * @boolean { false } syn-link--disabled Applies the disabled link styling
+ */
+.syn-link {
+  /* The size of the icon */
+  --syn-link-icon-size: var(--syn-spacing-large);
+
+  /* The gap that is used when there are pre or suffix icons */
+  --syn-link-icon-gap: var(--syn-spacing-x-small);
```

Comment:
On inheritat, we need something that uses em, as otherwise it maybe doesn't fit together.

Reply:
Inherit should use the latest font-size defined by an upper element, I do not see any issues with using it here? See https://codepen.io/schilchSICKAG/pen/PoraZyg for a demo.

Reply:
![CleanShot 2024-08-26 at 08 05 46@2x](https://github.com/user-attachments/assets/85705103-8fda-410a-a08a-2887891b0273)

The icons in inherit have to adapt proportionally as well. Otherwise it will look like... this.

Reply:
Ah, now I see what you mean :). Fixed in latest version. I adjusted both default values to `1em` to make it more explicit. I have played around with relative sizes (e.g. a default of `1.2em`), but that does not work out any time as icon sizes for defined size properties are not strictly aligned. Thanks for the catch.